src/pyramid/config/routes.py

```diff
@@ -139,6 +140,26 @@ def add_route(
 
           .. versionadded:: 1.1
 
+        inherit_slash
+
+          This argument can only be used when the ``pattern`` is an empty
+          string (``''``). By default, the composed route pattern will always
+          includes a trailing slash, but this argument provides a way to
+          opt-out if both, you and the integrator (the developer setting the
+          :term:`route prefix`), agree that the pattern should not contain
+          a trailing slash. For example:
+
+          .. code-block:: python
+
+              with config.route_prefix_context('/users'):
+                  config.add_route('users', '', inherit_slash=True)
+
+          In this example, the resulting route pattern will be ``/users``.
+          Alternatively, if the route prefix were ``/users/`` then the
```

Comment:
Comma placement:

```
Alternatively if the route prefix were ``/users/``, then the
```

Reply:
I kept the comma after alternatively, I'm pretty sure it's valid here.

Reply:
Both usages are valid. It's a modern style preference in English to use fewer commas. Commas should be used as "speed bumps", where you want the reader to pause. It's similar to the [breath mark in music](https://www.symbols.com/symbol/breath-mark).

Reply:
ok, I'm going to keep it the way it is